ci-operator/step-registry/cert-manager/custom-apiserver-cert/cert-manager-custom-apiserver-cert-commands.sh

```diff
@@ -164,21 +164,21 @@ spec:
         name: cert-manager-managed-alt-api-tls
 "
 
-# Wait for the clusteroperator kube-apiserver to finish rollout
+# Wait for the clusteroperator kube-apiserver to start rollout
 # Note, if $NEW_API_FQDN is $ORIGINAL_API_FQDN other than an alternative FQDN, all oc commands afterwards need to add the --insecure-skip-tls-verify flag before the KUBECONFIG is updated later
 MAX_RETRY=20
 INTERVAL=10
 COUNTER=0
 while :;
 do
-    echo "Checking clusteroperator kube-apiserver Progressing status for the #${COUNTER}-th time ..."
+    echo "Checking if clusteroperator kube-apiserver rollout has started for the #${COUNTER}-th time ..."
     if [ "$(oc get clusteroperator kube-apiserver -o jsonpath='{.status.conditions[?(@.type=="Progressing")].status}')" == True ]; then
-        echo "The clusteroperator kube-apiserver Progressing status becomes True." && break
+        echo "The clusteroperator kube-apiserver Progressing status becomes True, indicates rollout has started." && break
     fi
     ((++COUNTER))
     if [[ $COUNTER -eq $MAX_RETRY ]]; then
-        echo "The clusteroperator kube-apiserver Progressing status is not True after $((MAX_RETRY * INTERVAL)) seconds. Dumping status:"
-        oc get clusteroperator kube-apiserver -o jsonpath='{.status.conditions[?(@.type=="Progressing")]}'
+        echo "The clusteroperator kube-apiserver rollout did not start after $((MAX_RETRY * INTERVAL)) seconds. Dumping status:"
+        oc get clusteroperator kube-apiserver -o=jsonpath='{.status}'
```

Comment:
It does not harm to print longer full .status. But actually, indeed only the Progressing condition is relevant, even if for debugging.